designs/virtualization/4.7/add-ip-addresses/add-ip-addresses.md

```diff
@@ -0,0 +1,34 @@
+---
+parent: Virtualization
+version: 4.7
+---
+# Add V4 and V6 IP Addresses
+
+## Adding a ‘Last restored’ column to the list view
+
+Currently we only show IPv4 IP addresses but with the addition of IPv6 we should consider how these strings should surface in the list/details view of the virtual machine.
+Currently we would not be able to show both types together though.
+
+## List view
+
+The column header will change to IP address(es) to match multiple addresses.
+
+![List view](img/VM-list.png)
```

Comment:
sorry for the nit - but just to avoid confusion the "Create virtual machine" should just be a "Create" dropdown

Reply:
+1